default/AI/ColonisationAI.py

```diff
@@ -980,13 +993,16 @@ def evaluate_planet(planet_id, mission_type, spec_name, empire, detail=None):
         if have_existing_presence:
             detail.append("preexisting system colony")
             retval *= 1.5
+        supplyval = 0
         if sys_supply < 0:
             if sys_supply + planet_supply >= 0:
-                retval += 30 * (planet_supply - max(-3, sys_supply))
+                supplyval += 30 * (planet_supply - max(-3, sys_supply))
```

Comment:
I take it you are referring to the fact that when I print it to log the label I use is "supply_val" not "supplyval"?  I don't always use the exact variable name as the label in a logging statement, although I suppose that the fact that I used a slightly different label here suggests that perhaps the variable name itself would be more clear as supply_val.